sockeye/train.py

```diff
@@ -908,4 +908,6 @@ def train(args: argparse.Namespace) -> training.TrainState:
         return training_state
 
 if __name__ == "__main__":
+    if any([a == "--checkpoint-frequency" or a.startswith("--checkpoint-frequency=") for a in sys.argv]):
```

Comment:
I would vote for removing this in favor of my suggestion above.

Reply:
Does `SUPPRESS` produce some kind of warning message? If not, I'd still have something around to make users aware of the deprecation.

Reply:
I dont think it does. Is there a better way than checking sys.argv?

Reply:
I think the main alternative would be checking the parser object itself when instantiated for training: this was my initial impulse.  I went with checking sys.argv so it wouldn't be nested deeper than the training script (since it's a stop-gap) but maybe that wasn't a good reason.  I'll try the SUPPRESS approach, and see if I can still have the old usage trigger the warning, hopefully w/o the sys.argv string comparisons.